addons/web_tour/static/src/js/tour_service.js

```diff
@@ -36,6 +36,12 @@ return session.is_bound.then(function () {
         const disabled = session.tour_disable || device.isMobile;
         var tour_manager = new TourManager(rootWidget, consumed_tours, disabled);
 
+        const iframeContainer = 'wysiwyg_iframe';
```

Comment:
That variable name is poorly chosen :grimacing:

Reply:
Maybe it would be clearer that is a class with `iframeContainerClass` ?